docs/community-version/self-deploy/index.md

```diff
@@ -57,14 +58,23 @@ a13f349fe35b   minio/minio:RELEASE.2025-01-20T14-49-07Z   "/usr/bin/docker-ent
 e7b398dbd02b   postgres:16-alpine                         "docker-entrypoint.s…"   5 hours ago   Up 5 hours (healthy)   0.0.0.0:35432->5432/tcp, :::35432->5432/tcp                                              refly_db
 ```
 
-Finally, you can access the Refly application in `http://${HOST_IP}:5700`, where `${HOST_IP}` is the IP address of the host machine.
+### Deploy to Kubernetes cluster
```

Comment:
Could you also add this to the Chinese translation in `docs/zh/community-version/self-deploy/index.md`?